lib/shoryuken/subscriber.rb

```diff
@@ -0,0 +1,43 @@
+module Shoryuken
```

Comment:
Shoryuken support seems outside the scope of this gem.

Reply:
If you are going to add it, add something to the wiki or readme about support and add specs

Reply:
Yeah, I was going back and forth on it. I wanted to eliminate a lot of the boilerplate code we'd add though. I originally created another gem `aws-broker-rails` for both Shoryuken + aws-broker support but that seemed overkill for what we're trying to do.

I've added notes in the [readme](https://github.com/Thanx/aws-broker#rails-usage) and will add more notes in the [wiki](https://github.com/Thanx/aws-broker/wiki#shoryuken-support)

Reply:
👍 on specs - will add

Reply:
Main reason I wanted it as part of the gem was because we'd be using this interface and the other nucleus-dependent projects we have. Another way is we can move the Shoryuken::Subscriber to Nucleus. Thoughts?

Reply:
Talked over slack - going to move the Shoryuken support to nucleus